gateway/src/api/latest.rs

```diff
@@ -619,36 +616,6 @@ async fn request_custom_domain_acme_certificate(
         .create_custom_domain_certificate(&fqdn, &acme_client, &project_name, credentials)
         .await?;
 
-    let project = service.find_project(&project_name).await?;
-    let project_id = project
-        .state
-        .container()
-        .unwrap()
-        .project_id()
-        .map_err(|_| Error::custom(ErrorKind::Internal, "Missing project_id from the container"))?;
-
-    let container = project.state.container().unwrap();
-    let idle_minutes = container.idle_minutes();
-
-    // Destroy and recreate the project with the new domain.
-    service
-        .new_task()
-        .project(project_name.clone())
-        .and_then(task::destroy())
-        .and_then(task::run_until_done())
-        .and_then(task::run(move |ctx| async move {
-            let creating = ProjectCreating::new_with_random_initial_key(
```

Comment:
I have a feeling some of the methods called here (like `new_with_random_initial_key`) are now unused and their definitions can safely be removed

Reply:
Ctrl+F disagrees